packages/grafana-ui/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { FunctionComponent } from 'react';
+import { Range, createSliderWithTooltip } from 'rc-slider';
+import { cx, css } from 'emotion';
+import { Global, css as cssCore } from '@emotion/core';
+import { stylesFactory } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes/ThemeContext';
+import { Orientation } from '../../types/orientation';
+
+export interface Props {
+  min: number;
+  max: number;
+  orientation?: Orientation;
+  /** Set current positions of handle(s). If only 1 value supplied, only 1 handle displayed. */
+  value?: number[];
+  reverse?: boolean;
+  formatTooltipResult?: (value: number) => number | string;
+  onChange?: (values: number[]) => void;
+}
+
+const getStyles = stylesFactory((theme: GrafanaTheme, isHorizontal: boolean) => {
+  const trackColor = theme.isLight ? theme.colors.gray5 : theme.colors.dark6;
+  const container = isHorizontal
+    ? css`
+        width: 100%;
+        margin: ${theme.spacing.lg} ${theme.spacing.sm} ${theme.spacing.sm} ${theme.spacing.sm};
+      `
+    : css`
+        height: 100%;
+        margin: ${theme.spacing.sm} ${theme.spacing.lg} ${theme.spacing.sm} ${theme.spacing.sm};
+      `;
+
+  return {
+    container,
+    slider: css`
+      .rc-slider-vertical .rc-slider-handle {
+        margin-top: -10px;
+      }
+      .rc-slider-handle {
+        border: solid 2px ${theme.colors.blue77};
+        background-color: ${theme.colors.blue77};
+      }
+      .rc-slider-handle:hover {
+        border-color: ${theme.colors.blue77};
+      }
+      .rc-slider-handle:focus {
+        border-color: ${theme.colors.blue77};
+        box-shadow: none;
+      }
+      .rc-slider-handle:active {
+        border-color: ${theme.colors.blue77};
+        box-shadow: none;
+      }
+      .rc-slider-handle-click-focused:focus {
+        border-color: ${theme.colors.blue77};
+      }
+      .rc-slider-dot-active {
+        border-color: ${theme.colors.blue77};
+      }
+      .rc-slider-track {
+        background-color: ${theme.colors.blue77};
+      }
+      .rc-slider-rail {
+        background-color: ${trackColor};
+        border: 1px solid ${trackColor};
+      }
+    `,
+    tooltip: cssCore`
```

Comment:
@ivanahuckova please leave some comment about this usage in the code, ok? Same about `Global` below :)